common/src/main/java/gov/cms/ab2d/common/config/Mapping.java

```diff
@@ -15,14 +14,17 @@
 @Component
 public class Mapping {
 
-    @Autowired
-    private ContractService contractService;
+    private final ContractService contractService;
 
-    @Autowired
-    private RoleService roleService;
+    private final RoleService roleService;
 
     private ModelMapper modelMapper;
 
+    public Mapping(ContractService contractService, RoleService roleService) {
```

Comment:
Why can't lombok generate the constructor?

Reply:
Because it tried to inject the modelMapper which is generate in @PostConstruct